packages/datetime/src/dateInput.tsx

```diff
@@ -212,7 +211,7 @@ export class DateInput extends AbstractPureComponent2<IDateInputProps, IDateInpu
                 if (
                     e.key === "Tab" &&
                     !e.shiftKey &&
-                    this.lastTabbableElement.classList.contains(Classes.DATEPICKER_DAY)
+                    this.lastTabbableElement?.classList.contains(Classes.DATEPICKER_DAY)
```

Comment:
TBH, I'm not sure why the type checker wasn't catching this null call here. It might be worth doing a deeper pass of all current usage of refs to make sure they are properly null checked.

Reply:
not all packages use strict null checks, unfortunately. this one has that compiler option turned off at the moment.

Reply:
gotcha, makes sense. I'm so used to having that compiler turned on, I forgot TS doesn't check that by default haha